language/changes/7-packages.md

```diff
@@ -0,0 +1,325 @@
+* Easily build packages; and
+* Work with a common interface around compiled Move artifacts.
+
+## Motivation
+
+Until now, when writing Move you had no easy options of being able to
+package Move code into reusable chunks that could be used by others. There
+was no common compiled object model or on-disk representation of compiled
+Move code and its associated artifacts, e.g., source maps and generated
+documentation. And finally, there was no way of easily managing named
+addresses that spanned "logical groupings" of Move code.
+
+The Move package system aims to make these actions easier by providing a
+common compiled-object model and access to both compiled bytecode and
+  associated compilation artifacts along with ways to:
+* Declare logical packages of Move code;
+* Import and use packages;
+* Compile and generate associated compilation artifacts from packages; and
+* Manage--expose and instantiate--named addresses in packages.
+
+## Package Layout and Manifest Syntax
+
+A Move package source directory contains a `Move.toml` package manifest
+file along with a set of subdirectories:
+
+```
+a_move_package
+├── Move.toml      (required)
+├── sources        (required)
+├── examples       (optional, test & dev mode)
+├── scripts        (optional)
+├── doc_templates  (optional)
+└── tests          (optional, test mode)
+```
+
+The directories marked `required` _must_ be present in order for the directory
+to be considered a Move package and to be compiled. Optional directories can
+be present, and if so will be included in the compilation process. Depending on
+the mode that the package is built with (`test` or `dev`), the `tests` and
+`examples` directories will be included as well.
+
+The `sources` directory can contain both Move modules and Move scripts (both
+transaction scripts and modules containing script functions). The `examples`
+directory can hold additional code to be used only for development and/or
+tutorial purposes that will not be included when compiled outside `test` or
+`dev` mode.
+
+Move code in the `scripts` directory will be included if present.
+Documentation will be built using any [documentation
+templates](../move-prover/doc/user/docgen.md) present in the `doc_templates`
+directory.
+
+### Move.toml
+
+The Move package manifest is defined within the `Move.toml` file and has the
+following syntax. Optional fields are marked with `*`, `+` denotes
+one or more elements:
+
+```
+[package]
+name = <string>                  # e.g., "MoveStdlib"
+version = "<uint>.<uint>.<uint>" # e.g., "0.1.1"
+license* = <string>              # e.g., "MIT", "GPL", "Apache 2.0"
+authors* = [<string>]            # e.g., ["Joe Smith (joesmith@noemail.com)", "Jane Smith (janesmith@noemail.com)"]
+
+[addresses]  # (Optional section) Declares named addresses in this package and instantiates named addresses in the package graph
+<addr_name> = "_" | "<hex_address>" # e.g., Std = "_"
+...
```

Comment:
I'm not sure about this "..." followed by another line approach. Isn't this needed for each of the sections? An alternative might be to add a comment before the line, saying something like:
`# One or more lines declaring named addresses in the following format:`

Reply:
But whatever approach you take, I think it's needed for each section

Reply:
Went for the `# One or more ...` comments approach